src/app/main-nav/main-nav.component.scss

```diff
@@ -0,0 +1,6 @@
+@media (max-width: 40.0525em) {
```

Comment:
why does the max-width have to be so specific...? would be nice if this value could live in a variable and then reused

Reply:
done